sphinx_pyscript/__init__.py

```diff
@@ -72,52 +78,66 @@ def run(self):
             code = "\n".join(self.content)
         else:
             raise self.error("Must provide either content or the 'file' option")
-        return [nodes.raw("", f"<py-script>\n{code}\n</py-script>\n", format="html")]
+        return [
+            nodes.raw("", f"<script type='py'>\n{code}\n</script>\n", format="html")
+        ]
 
 
-class PyRepl(SphinxDirective):
-    """Add a py-repl tag"""
+class PyEditor(SphinxDirective):
+    """Add a py-editor tag"""
 
     has_content = True
     option_spec = {
-        "auto-generate": directives.flag,
-        "output": directives.unchanged,
+        "setup": directives.flag,
+        "env": directives.unchanged,
+        "config": directives.unchanged,
```

Comment:
it would be good if you could but a small comment on how each of these are intended to be used (or even even better add a section in the docs)

Reply:
Added comments and a section in the docs, both good ideas!